proteuslib/unit_models/reverse_osmosis_0D.py

```diff
@@ -310,40 +309,40 @@ def build(self):
                 self.io_list,
                 self.solute_list,
                 initialize=1e-5,
-                bounds=(1e-10, 1),
+                bounds=(1e-6, 1),
                 domain=NonNegativeReals,
                 units=units_meta('length') * units_meta('time')**-1,
                 doc='Mass transfer coefficient in feed channel at inlet and outlet')
         if ((self.config.mass_transfer_coefficient == MassTransferCoefficient.calculated)
                 or self.config.pressure_change_type == PressureChangeType.calculated):
-            self.width = Var(
-                initialize=1,
-                bounds=(1e-8, 1e6),
-                domain=NonNegativeReals,
-                units=units_meta('length'),
-                doc='Effective membrane width')
             self.channel_height = Var(
-                initialize=7.5e-4,
+                initialize=1e-3,
                 bounds=(1e-8, 1),
                 domain=NonNegativeReals,
                 units=units_meta('length'),
                 doc='Feed-channel height')
+            self.dh = Var(
+                initialize=1e-3,
+                bounds=(1e-5, 1e-2),
+                domain=NonNegativeReals,
+                units=units_meta('length'),
+                doc='Hydraulic diameter of feed channel')
+            self.df = Var(
```

Comment:
I thought we discussed making df either a calculated variable (based on a constraint) or substituting it out with df = 1/2 h